src/Samples/Multiselect/MainWindow.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<Window x:Class="Elmish.WPF.Samples.Multiselect.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:vm="clr-namespace:Elmish.WPF.Samples.Multiselect;assembly=Multiselect.Core"
+        xmlns:sys="clr-namespace:System;assembly=mscorlib"
+        WindowStartupLocation="CenterScreen"
+        Title="SubModelSelectedItem"
+        Height="350"
+        Width="500"
+        mc:Ignorable="d"
+        d:DataContext="{x:Static vm:Program.designVm}">
```

Comment:
I have some problem in my VS with this declaration but as far as I can tell, it's exactly defined as in the other projects were I don't have an issue.